### PR TITLE
[16.0][MIG] project_hr: Migration to version 16.0

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
cc @Tecnativa TT43237

@pedrobaeza @ernestotejeda please review

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

**Review by CarlosRoca13 — APPROVED** [review]

File: project_hr/models/project_task.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        help="Technical field for computing allowed users according employee "
+        "category.",
+    )
+
+    @api.depends("user_ids", "company_id")
+    def _compute_employee_ids(self):
+        for task in self.filtered("user_ids"):
+            task.employee_ids = task.user_ids.employee_ids.filtered(
+                lambda x: x.company_id == task.company_id
+            )[:1]
```
Comment (ernestotejeda) [inline project_hr/models/project_task.py]:
```suggestion
            )
```

I think you should remove this

**Review by ernestotejeda — CHANGES_REQUESTED** [review]

File: project_hr/models/project_task.py (hunk 1 above)
Comment (pilarvargas-tecnativa) [reply project_hr/models/project_task.py]: thanks :)

**Review by ernestotejeda — APPROVED** [review]

**Pull request merged**